packages/contracts/scripts/engine/V3/post-batch-create-engine-contracts.ts

```diff
@@ -183,12 +183,16 @@ async function main() {
 
       // create image bucket
       let imageBucketCreated = false;
+      // For studio contracts use token ticker as bucket name, otherwise use token
```

Comment:
Note - Updated the bucket naming logic slightly so that studio contracts use the token ticker instead of the token name, alternatively we could remove the pipe symbol in the logic, I'm open to either! 🙏